src/main/java/com/google/codeu/data/Datastore.java

```diff
@@ -61,16 +65,20 @@ public void storeMessage(Message message) {
             .addSort("timestamp", SortDirection.DESCENDING);
     PreparedQuery results = datastore.prepare(query);
 
-    for (Entity entity : results.asIterable()) {
-      try {
+    for (Entity entity : results.asIterable())
+    {
+      try
+      {
         String idString = entity.getKey().getName();
         UUID id = UUID.fromString(idString);
+        String recipient = (String) entity.getProperty("recipient"); //I'll probably figure out what this does later
         String text = (String) entity.getProperty("text");
         long timestamp = (long) entity.getProperty("timestamp");
 
-        Message message = new Message(id, user, text, timestamp);
+        Message message = new Message(id, user, text, timestamp, recipient); //added recipient to the parameters
```

Comment:
Remove comment. Remember other folks will be reading your comments in the codebase (even those who have not reviewed your code). If everyone added a comment everytime they added a new parameter, this line would be very long. It is assumed that recipient is now part of Message.